fab/fabfile.py

```diff
@@ -367,6 +367,14 @@ def install_packages():
 
 
 @task
+@roles(ROLES_TOUCHFORMS)
+def install_npm_packages():
+    """Install required NPM packages for server"""
+    with cd(os.path.join(env.code_root, 'submodules/touchforms-src/touchforms')):
+        sudo("npm install")
```

Comment:
it's only run once (not part of the main deploy). so once it's been successfully installed, we won't have to worry about it until someone wants to update the packages. i don't necessarily like the idea of hosting our own. i think that's way too much work to maintain for such a little component. any thoughts @dannyroberts?